hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/FederationUtil.java

```diff
@@ -98,13 +98,10 @@ public static JSONArray getJmx(String beanQuery, String webAddress,
       JSONObject json = new JSONObject(jmxOutput);
       ret = json.getJSONArray("beans");
     } catch (IOException e) {
-      LOG.error("Cannot read JMX bean {} from server {}: {}",
-          beanQuery, webAddress, e.getMessage());
-    } catch (JSONException e) {
-      LOG.error("Cannot parse JMX output for {} from server {}: {}",
-          beanQuery, webAddress, e.getMessage());
+      LOG.error("Cannot read JMX bean {} from server {}: ",
```

Comment:
Leave as:
LOG.error("Cannot parse JMX output for {} from server {}",
beanQuery, webAddress, e);